.github/workflows/test.yml

```diff
@@ -48,6 +48,8 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@stable
+      - run: cargo test -p tendermint-proto
+      - run: cargo test -p tendermint-proto --features std
```

Comment:
These caught the failure: https://github.com/informalsystems/tendermint-rs/actions/runs/10290702236/job/28481276076?pr=1455

Reply:
I've added `cargo hack test --each-feature` to the test suite to ensure we test each feature independently, which caught a few more errors in other crates as well.